BUILD.gn

```diff
@@ -1454,6 +1467,16 @@ group("electron_mksnapshot") {
 
 dist_zip("electron_mksnapshot_zip") {
   data_deps = mksnapshot_deps
+  if (is_linux && is_official_build) {
+    data_deps += [
+      ":strip_libEGL_shlib",
+      ":strip_libGLESv2_shlib",
+      ":strip_libffmpeg_shlib",
+      ":strip_libvk_swiftshader_shlib",
```

Comment:
Why are we shipping these libraries as part of mksnapshot package ? This is not new in this PR, seems like we have been doing this for a while. These are chromium specific and mksnapshot should not be aware of them.

Reply:
@deepak1556, it looks like these are getting pulled in because the contents of the zip gets pulled from the GN runtime deps for the target. These come in via the dep chain:

```
//electron:electron_mksnapshot_zip -->
//tools/v8_context_snapshot:v8_context_snapshot_generator -->
//third_party/blink/public:blink
```

Reply:
I see, are the GL files also linked in the default snapshot ? I didn't see anything in the dynamic section requiring the GL libraries.